pkgs/applications/networking/protonvpn-cli/default.nix

```diff
@@ -28,12 +28,9 @@ in stdenv.mkDerivation rec {
         "  echo \"Connecting...\"" \
         "  sed -ri 's@${expectedUpdateResolvPath}@${actualUpdateResolvePath}@g' \"\$openvpn_config\"; echo \"Connecting...\""
     cp protonvpn-cli.sh "$out/bin/protonvpn-cli"
-    ln -s "$out/bin/protonvpn-cli" "$out/bin/pvpn"
-  '';
-
-  postInstallPhase = ''
```

Comment:
this didn't work because `postInstallPhase` isn't a phase in stdenv, and we've overidden the default `installPhase`. so that means it doesn't have `runHook preInstall` which runs it.